src/contrib/persistence/Akka.Persistence.Sqlite.Tests/SqliteJournalConnectionFailureSpec.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Persistence.Sql.TestKit;
+using Akka.Util.Internal;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Sqlite.Tests
+{
+    public class SqliteJournalConnectionFailureSpec : SqlJournalConnectionFailureSpec
```

Comment:
An example of inheriting the base `SqlJournalConnectionFailureSpec` to make sure a `SqliteJournal` given a bad connection string will throw an error clearly to the user